src/packagedcode/pypi.py

```diff
@@ -2281,7 +2286,7 @@ def get_pypi_urls(name, version, **kwargs):
     )
 
 
-def get_urls(metainfo, name, version, poetry=False):
+def get_urls(metainfo, name, version,is_private, poetry=False):
```

Comment:
```suggestion
def get_urls(metainfo, name, version, is_private=False, poetry=False):
```

If you don't add defaults to a function you just added, this will fail at all the places this function is being invoked other than in the `PyprojectTomlHandler.parse()` function.

And you can see the test failures at https://dev.azure.com/nexB/scancode-toolkit/_build/results?buildId=15878&view=logs&j=1412dbfa-421a-5a40-8ddf-7073fc46aa19

You should run all the tests for a module you are changing locally before you push any code, making sure they pass. In this case tests/packagedcode/test_pypi.py